### ISSUE TITLE
Formats only work on first line

### ISSUE THREAD

**Issue by Remo** [CroudSupport/vue-quill#8]
I'm having a strange issue, tried to debug it for a while, but didn't get much smarter yet.

When I want to add a bullet list, the editor always ignored my selection and simply converts the first line into a bullet list. I tried to disable the focusEditor method, but no luck.

Any idea where to search for the cause of this? I'm happy to debug, but I'd appreciate some pointers in the right direction.

**Comment by Remo** [comment]
If I remove `v-el:toolbar` and `v-el:quill` and use hard coded ID's instead it works. I'm on vue.js 1.0.26 if that helps.
I don't want to use ID's though...

**Comment by BrockReece** [comment]
Hi @Remo, 

Just for clarity, you have multiple lines of text in a quill editor, you are highlighting all the lines, pressing the bullet list button in the toolbar but only the first line is being formatted? 
What version of quill are you running?

Cheers
Brock

**Comment by Remo** [comment]
@BrockReece thanks for keeping an eye on this! I don't have it on this computer, but I ran `npm install` today, I guess it should be the current version linked by this repo. No, I have two lines, select only the second and when I hit the button it changes the first line. It's as if the text selection is always reset to the first character.

**Comment by Remo** [comment]
I'm using quill 0.20.1

**Comment by BrockReece** [comment]
Oh I see, thanks. 
I will have a look into it this morning

**Comment by BrockReece** [comment]
Hi @Remo 
I too am using the quill 0.20.1 but I can't replicate the problem that you are having?

**Comment by Remo** [comment]
It happens on a fairly large project with vue-router and so on. Maybe there's a conflict with something else. I'll try to create a new project from scratch..

**Comment by BrockReece** [comment]
It could be, I wish I could help and would be very interested to know if you figure anything out.

**Comment by Remo** [comment]
Okay, had to add quite a few things to get it working. This commit causes the problem
https://github.com/Remo/vue-test-quill/commit/62d9ca6c800f1fa7d953719a625e620a6f5212db

I was lazy and added an old version of vue-strap as I had lots of problems with the current version. 
I then updated to the latest stable release https://github.com/Remo/vue-test-quill/commit/7bb5994bf2b6d56d0ec3e72f3eed2ccd7e7c8190, and run into the problem which made me use that old version in the first place:

```
node_modules/vue-strap/src/spinner.scss:10
$brand-primary: darken(#428bca, 6.5%); // #337ab7
              ^
ParseError: Unexpected token
[16:03:05] Finished 'browserify' after 5.82 s
[16:03:05] Starting 'version'...
[16:03:06] Finished 'version' after 360 ms
[16:03:06] Finished 'all' after 6.18 s
[16:03:06] Starting 'default'...
```

I then deleted the content of `spinner.scss` to get everything working, because who cares about CSS.

Running `gulp` again and still the same problem.

It seems to be a combination of vue-strap and vue-quill, but not sure who's fault it actually is, I'd assume vue-strap.
